server/src/services/organizations.service.ts

```diff
@@ -112,10 +112,43 @@ export class OrganizationsService {
     }, manager);
   }
 
-  async fetchUsers(user: any): Promise<FetchUserResponse[]> {
+  async fetchUsersByValue(user: any, searchInput: string): Promise<any> {
+    const organizationUsers = await createQueryBuilder(OrganizationUser, 'organization_user')
+      .innerJoinAndSelect('organization_user.user', 'user')
+      .where('organization_user.organization_id = :organizationId', {
+        organizationId: user.organizationId,
+      })
+      .andWhere(
+        new Brackets((qb) => {
+          qb.where('lower(user.email) like :email', {
+            email: `%${searchInput.toLowerCase()}%`,
+          })
+            .orWhere('lower(user.firstName) like :firstName', {
+              firstName: `%${searchInput.toLowerCase()}%`,
+            })
+            .orWhere('lower(user.lastName) like :lastName', { lastName: `%${searchInput.toLowerCase()}%` });
+        })
+      )
+      .getMany();
+
+    return organizationUsers?.map((orgUser) => {
+      return {
+        email: orgUser.user.email,
+        firstName: orgUser.user.firstName,
+        lastName: orgUser.user.lastName,
+        name: `${orgUser.user.firstName} ${orgUser.user.lastName}`,
+        id: orgUser.id,
+        userId: orgUser.user.id,
+      };
+    });
+  }
+
+  async fetchUsers(user: any, page: number, options: any): Promise<FetchUserResponse[]> {
     const organizationUsers = await this.organizationUsersRepository.find({
-      where: { organizationId: user.organizationId },
+      where: { organizationId: user.organizationId, user: options },
```

Comment:
Shall we add wildcard search here?

Reply:
<img width="1285" alt="Screenshot 2022-09-08 at 10 18 45 AM" src="https://user-images.githubusercontent.com/3417097/189042444-23bc2c1c-ca98-470f-b98f-2c7b953b1686.png">
and a no results found message